src/components/modals/RateDialogsModal.tsx

```diff
@@ -28,26 +29,35 @@ interface ComponentState {
     ratingPair: Test.RatingPair | undefined
     resultIndex: number
     webchatKey: number,
-    activities1: BotChat.Activity[] | undefined
-    activities2: BotChat.Activity[] | undefined
+    activities1: BB.Activity[] | undefined
```

Comment:
Why do we sometimes use activity from the custom webchat and somtimes bot builder? I would expect it to always come from bot builder?

Reply:
Working on switching all Activity refs over to BB.